src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1535,4 +1535,7 @@ I agree to all of the foregoing:</value>
   <data name="Bitness64" xml:space="preserve">
     <value>64-bit</value>
   </data>
+  <data name="Use_64bit_process_for_code_analysis" xml:space="preserve">
+    <value>Use 64-bit process for code analysis (requires restart)</value>
```

Comment:
Can you make the data name match the text (including 'requires restart')

Reply:
➡️ Done